cves/kernel/CVE-2013-2094.yml

```diff
@@ -116,362 +29,104 @@ upvotes_instructions: |
   upvotes score on your branch.
 upvotes:
 unit_tested:
-  question: |
-    Were automated unit tests involved in this vulnerability?
-    Was the original code unit tested, or not unit tested? Did the fix involve
-    improving the automated tests?
-
-    For code: and fix: - your answer should be boolean.
-
-    For the code_answer below, look not only at the fix but the surrounding
-    code near the fix in related directories and determine if and was there were
-    unit tests involved for this subsystem.
-
-    For the fix_answer below, check if the fix for the vulnerability involves
-    adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: This was not unit tested
+  fix: false
+  fix_answer: This was not unit tested
 discovered:
-  question: |
-    How was this vulnerability discovered?
-
-    Go to the bug report and read the conversation to find out how this was
-    originally found. Answer in longform below in "answer", fill in the date in
-    YYYY-MM-DD, and then determine if the vulnerability was found by a Google
-    employee (you can tell from their email address). If it's clear that the
-    vulenrability was discovered by a contest, fill in the name there.
-
-    The automated, contest, and developer flags can be true, false, or nil.
-
-    If there is no evidence as to how this vulnerability was found, then please
-    explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    Founf by Tommi Rantala at University of Indiana on 2013-04-12
+    https://lkml.indiana.edu/hypermail/linux/kernel/1304.1/03652.html
+  automated: false
```

Comment:
Small typo here. Also, this is one of the few vulnerabilities I've seen so far that discusses how it was discovered.